### PR TITLE
Marc edits

### PR DESCRIPTION
**Pull request by howdym**
It's just an extension. Nothing removed, only added.

### CODE REVIEW

File: contact/index.html [hunk 1]
```diff
@@ -57,19 +57,19 @@ <h1 role="banner">technological innovations for inclusive learning &amp; teachin
     <section class="textblock">
       <h1 class="window__header">Contact</h1>
       <ul class="list_with_padding">
-        <li>email us at <a href="mailto:tiiltlab@gmail.com">tiiltlab@gmail.com</a></li>
-        <li>follow us on <a href="https://twitter.com/tiiltlab">twitter</a></li>
-        <li><a  target="_blank" href="https://www.google.com/maps/place/Mudd+Library/@42.0581472,-87.6765888,17z/data=!3m1!4b1!4m5!3m4!1s0x880fda9eebe51fd3:0x303e4571ee131fdd!8m2!3d42.0581472!4d-87.6744001">visit
+        <li>email us at <a href="mailto:tiiltlab@gmail.com" aria-label="Our Gmail">tiiltlab@gmail.com</a></li>
+        <li>follow us on <a href="https://twitter.com/tiiltlab" aria-label="Our Twitter">twitter</a></li>
+        <li><a target="_blank" href="https://www.google.com/maps/place/Mudd+Library/@42.0581472,-87.6765888,17z/data=!3m1!4b1!4m5!3m4!1s0x880fda9eebe51fd3:0x303e4571ee131fdd!8m2!3d42.0581472!4d-87.6744001" aria-label="Our Address">visit
```
Comment (klhmn5) [inline contact/index.html]: Can we make this more specific? Like location of the lab or something. Address sometimes can refer to different things. And also makes sure people know it isn't Marcelo's office

File: projects/index.html [hunk 2]
```diff
@@ -72,6 +72,7 @@ <h1>Learning with Minecraft</h1>
       </p>
     </section>
     <section class="textblock project_box">
+      <!-- <span aria-haspopup="true">▼</span> Should this be added? -->
```
Comment (klhmn5) [inline projects/index.html]: Is it a pop-up or an expanded? It is currently just revealing an image

File: templates.html [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <!-- PROJECTS -->
 <section class="textblock project_box">
-        <h1><span>▼</span></h1>
-        <img class="lazy visually_hidden" src="">
+        <h1><span tabindex=0 aria-haspopup="true" aria-expanded="false">▼</span></h1>
+        <img alt="Refer to WAI Accessibility Guildlines on how to fill this out." class="lazy visually_hidden" src="">
```
Comment (klhmn5) [inline templates.html:4]: Is this alt text a note for us, or is It currently written to read to a screen reader?

Comment (howdym) [reply templates.html:4]: Both. It's a note to us on how to fill it out for the screen reader to read.

**Pull request merged**